extensions/context/src/grpcInOtelTest/java/io/grpc/override/ContextStorageOverride.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.override;
+
+import io.grpc.Context;
+import io.opentelemetry.context.ContextKey;
+import io.opentelemetry.context.Scope;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class ContextStorageOverride extends Context.Storage {
```

Comment:
I might be missing something, but I don't see this being used anywhere.

Reply:
Neve rmind. I read the grpc docs and see how this works. Let's add an annotation so that IDEA doesn't suggest deleting it. :)

Reply:
interesting. IDEA was marking it as unused for me. :hmmm: